taprpc/taprootassets.proto

```diff
@@ -611,6 +611,23 @@ message ListTransfersResponse {
     repeated AssetTransfer transfers = 1;
 }
 
+// BlockHash represents a hash of a Bitcoin (BTC) blockchain block.
+//
+// This message includes two separate fields for the same hash:
+// - The `hash` field provides the raw byte format, which is convenient for Go
+//   developers to use directly in other RPC calls.
+// - The `reversed_hash_hex` field provides the byte-reversed hash in
+//   hexadecimal string format, which is useful for users who need to copy and
+//   paste the value, such as passing it as a command-line argument.
+message BlockHash {
+    // The block hash in raw byte format.
+    bytes hash = 1;
+
+    // The block hash as a hexadecimal string of the byte-reversed hash,
+    // commonly used in Bitcoin for human-readable representation.
+    string reversed_hash_hex = 2;
```

Comment:
The convention for naming this would just be `hash_str` both in other projects and this one:
 - https://github.com/lightningnetwork/lnd/blob/5e84ba92afe4cd13d1360fb76eb751ec328ee1b5/lnrpc/lightning.proto#L1070
 - https://github.com/lightningnetwork/lnd/blob/5e84ba92afe4cd13d1360fb76eb751ec328ee1b5/lnrpc/lightning.proto#L1058
 - https://github.com/lightninglabs/taproot-assets/blob/faea364ddc0a1c9359fadfd9b9b3570ed7a727c7/taprpc/assetwalletrpc/assetwallet.proto#L424
 - https://github.com/lightninglabs/taproot-assets/blob/2f369531aab60b141a8ec8fb8205574ec6412a9f/taprpc/universerpc/universe.proto#L279

We should aim for some consistency across our repositories to make developers' lives easier. The comment above it should be explanation enough.